src/wwwroot/partials/settings/my-plan.html

```diff
@@ -41,7 +41,7 @@
       </div>
       <div class="flex two flex--wrap">
         <label class="width--full special uppercase">{{ 'my_plan_renewal_date' | translate }}</label>
-        <p class="renewal-date">{{ vm.resetDate | amUtc | amDateFormat:'YYYY-MM-DD HH:mm Z' }}</p>
+        <p class="renewal-date">{{ vm.resetDate | date :'mediumDate' :'+0' }}</p>
```

Comment:
I think that the black space between `date ` and ` :` is not correct, do you think?